course/urls.py

```diff
@@ -16,6 +16,12 @@
     url(r'^archive/$',
         views.ArchiveView.as_view(),
         name="archive"),
+    url(COURSE_URL_PREFIX + r'instances/$',
+        views.CourseInstancesView.as_view(),
+        name="course_instances"),
+    url(COURSE_URL_PREFIX + r'$',
+        views.LastInstanceView.as_view(),
+        name="course-no-instance"),
```

Comment:
Consistent naming is always good to have. Many urls have names with underscores `_`, but this one uses hyphens `-`. Shall we use underscores here for the sake of consistency?